flink-streaming-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaConsumerTestBase.java

```diff
@@ -186,7 +190,7 @@ public void runFailOnNoBrokerTest() throws Exception {
 			stream.print();
 			see.execute("No broker test");
 		} catch(RuntimeException re) {
-			if(kafkaServer.getVersion().equals("0.9")) {
+			if(kafkaServer.getVersion().equals(getExpectedKafkaVersion())) {
```

Comment:
I think the original intent of this assert here was that 0.8 connector throws different exception messages than 0.9.
So adding the `getExpectedKafkaVersion()` is a bit confusing with respect to the test intent.
Perhaps we should remove the new `getExpectedKafkaVersion()` and simply change this to `kafkaServer.getVersion().equals("0.9") || kafkaServer.getVersion().equals("0.10")`?